### ISSUE TITLE
Inductor Error: timm models fp16 inference failed

### ISSUE THREAD

**Issue by pyjhzwh** [pytorch/torchdynamo#929]
On main branch (commit 08aa6cea0673fa97e43f5fa5ec09588e0af23d1b)
`python benchmarks/torchbench.py --inductor-settings -dcuda --float16 -n50 --inductor --filter timm_efficientdet` failed

```
Traceback (most recent call last):                                                                                                                                                            
  File "/scratch/panyj/work/cpy/torchdynamo/benchmarks/torchbench.py", line 363, in <module>                                                                                                  
    main(TorchBenchmarkRunner(), original_dir)                                                                                                                                                
  File "/scratch/panyj/work/cpy/torchdynamo/benchmarks/common.py", line 1708, in main                                                                                                         
    device, name, model, example_inputs, batch_size = runner.load_model(                                                                                                                      
  File "/scratch/panyj/work/cpy/torchdynamo/benchmarks/torchbench.py", line 256, in load_model                                                                                                
    module = importlib.import_module(f"torchbenchmark.models.{model_name}")                                                                                                                   
  File "/fsx/users/panyj/conda/envs/cpy/lib/python3.9/importlib/__init__.py", line 127, in import_module                                                                                      
    return _bootstrap._gcd_import(name[level:], package, level)                                                                                                                               
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import                                                                                                                             
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load                                                                                                                          
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked                                                                                                                  
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked                                                                                                                           
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module                                                                                                                     
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed                                                                                                                
  File "/scratch/panyj/work/torchbenchmark/torchbenchmark/models/timm_efficientdet/__init__.py", line 12, in <module>                                                                         
    from effdet import create_model, create_loader                                                                                                                                            
  File "/fsx/users/panyj/conda/envs/cpy/lib/python3.9/site-packages/effdet/__init__.py", line 2, in <module>                                                                                  
    from .bench import DetBenchPredict, DetBenchTrain, unwrap_bench                                                                                                                           
  File "/fsx/users/panyj/conda/envs/cpy/lib/python3.9/site-packages/effdet/bench.py", line 70, in <module>                                                                                    
    def _batch_detection(                                                                                                                                                                     
  File "/scratch/panyj/work/pytorch/torch/jit/_script.py", line 1343, in script                                                                                                               
    fn = torch._C._jit_script_compile(                                                                                                                                                        
  File "/scratch/panyj/work/pytorch/torch/jit/_recursive.py", line 863, in try_compile_fn                                                                                                     
    return torch.jit.script(fn, _rcb=rcb)                                                                                                                                                     
  File "/scratch/panyj/work/pytorch/torch/jit/_script.py", line 1343, in script                                                                                                               
    fn = torch._C._jit_script_compile(                                                                                                                                                        
  File "/scratch/panyj/work/pytorch/torch/jit/_recursive.py", line 863, in try_compile_fn                                                                                                     
    return torch.jit.script(fn, _rcb=rcb)                                                                                                                                                     
  File "/scratch/panyj/work/pytorch/torch/jit/_script.py", line 1343, in script                                                                                                               
    fn = torch._C._jit_script_compile(                                                                                                                                                        
  File "/scratch/panyj/work/pytorch/torch/jit/_recursive.py", line 863, in try_compile_fn                                                                                                     
    return torch.jit.script(fn, _rcb=rcb)                                                                                                                                                     
  File "/scratch/panyj/work/pytorch/torch/jit/_script.py", line 1343, in script                                                                                                               
    fn = torch._C._jit_script_compile(                                                                                                                                                        
  File "/scratch/panyj/work/pytorch/torch/jit/_recursive.py", line 863, in try_compile_fn                                                                                                     
    return torch.jit.script(fn, _rcb=rcb)                                                                                                                                                     
  File "/scratch/panyj/work/pytorch/torch/jit/_script.py", line 1343, in script                                                                                                               
    fn = torch._C._jit_script_compile(                                                                                                                                                        
RuntimeError:
object has no attribute nms:                                                                                                                                                                  
  File "/scratch/panyj/work/vision/torchvision/ops/boxes.py", line 41                                                                                                                         
        _log_api_usage_once(nms)                                                                                                                                                              
    _assert_has_ops()
    return torch.ops.torchvision.nms(boxes, scores, iou_threshold)
           ~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
'nms' is being compiled since it was called from '_batched_nms_vanilla'
  File "/scratch/panyj/work/vision/torchvision/ops/boxes.py", line 109
    for class_id in torch.unique(idxs):
        curr_indices = torch.where(idxs == class_id)[0]
        curr_keep_indices = nms(boxes[curr_indices], scores[curr_indices], iou_threshold)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
        keep_mask[curr_indices[curr_keep_indices]] = True
    keep_indices = torch.where(keep_mask)[0]
'_batched_nms_vanilla' is being compiled since it was called from 'batched_nms'
  File "/scratch/panyj/work/vision/torchvision/ops/boxes.py", line 73
    # https://github.com/pytorch/vision/issues/1311#issuecomment-781329339
    if boxes.numel() > (4000 if boxes.device.type == "cpu" else 20000) and not torchvision._is_tracing():
        return _batched_nms_vanilla(boxes, scores, idxs, iou_threshold)
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    else:
        return _batched_nms_coordinate_trick(boxes, scores, idxs, iou_threshold)
'batched_nms' is being compiled since it was called from 'generate_detections'
  File "/fsx/users/panyj/conda/envs/cpy/lib/python3.9/site-packages/effdet/anchors.py", line 140
        scores[top_detection_idx] = soft_scores
    else:
        top_detection_idx = batched_nms(boxes, scores, classes, iou_threshold=0.5)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE

    # keep only top max_det_per_image scoring predictions
'generate_detections' is being compiled since it was called from '_batch_detection'
  File "/fsx/users/panyj/conda/envs/cpy/lib/python3.9/site-packages/effdet/bench.py", line 82
        img_scale_i = None if img_scale is None else img_scale[i]
        img_size_i = None if img_size is None else img_size[i]
        detections = generate_detections(
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            class_out[i], box_out[i], anchor_boxes, indices[i], classes[i],
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            img_scale_i, img_size_i, max_det_per_image=max_det_per_image, soft_nms=soft_nms)
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
        batch_detections.append(detections)
    return torch.stack(batch_detections, dim=0)

```

Also, inference failed on "timm_efficientnet", "timm_nfnet", ... "timm_vovnet" models with error
```
TORCHDYNAMO: backend compiler failed
Traceback (most recent call last):
  File "/scratch/panyj/work/cpy/torchdynamo/torchdynamo/output_graph.py", line 383, in call_user_compiler
    compiled_fn = self.compiler_fn(gm, self.example_inputs())
  File "/scratch/panyj/work/cpy/torchdynamo/torchinductor/compile_fx.py", line 269, in compile_fx
    return compile_fx_aot(model_, example_inputs_)
  File "/scratch/panyj/work/cpy/torchdynamo/torchinductor/compile_fx.py", line 227, in compile_fx_aot
    model_ = normalize_ir(model_, example_inputs_)
  File "/scratch/panyj/work/cpy/torchdynamo/torchdynamo/optimizations/normalize.py", line 436, in normalize_ir
    normalize(gm)
  File "/scratch/panyj/work/cpy/torchdynamo/torchdynamo/optimizations/normalize.py", line 422, in normalize
    swap_node(
  File "/scratch/panyj/work/cpy/torchdynamo/torchdynamo/optimizations/normalize.py", line 401, in swap_node
    assert not new_node.meta, "New node must not have meta populated"
AssertionError: New node must not have meta populated
```

**Comment by Chillee** [comment]
> AssertionError: New node must not have meta populated

This error appears to be from https://github.com/pytorch/torchdynamo/pull/914/files

cc: @SherlockNoMad

**Comment by voznesenskym** [comment]
I think maybe instead of swapping meta, we should just swap the call stack component of meta? I think that was the intention of the original PR.

**Comment by jansel** [comment]
Fixed by #935
